EIPS/eip-5069.md

```diff
@@ -1,44 +1,131 @@
+
+ - A strong belief in the above mission;
+ - Proficiency with English (both written and spoken);
+ - Reading and critiquing EIPs;
+ - Participation in governance.
+
+EIP Editors are expected to meet these requirements throughout their tenure, and not doing so is grounds for removal. Any member may delegate some or all of their responsibilities/powers to tools and/or to other people.
+
+Membership requirements for Topic Groups are defined in their respective charters.
+
+## Making Decisions
+
+### Informally
+
+For decisions that are unlikely to be controversial&mdash;especially for decisions affecting a single proposal&mdash;an EIP Editor may choose whatever option they deem appropriate in accordance with our mission.
+
+### Formally
+
+Electing a Janitor, adding/removing EIP Editors, and any possibly-controversial decisions must all be made using variations of this formal process.
+
+#### Preparation
+
+##### Call for Input
+
+For any matter requiring a decision, a call for input must be published in writing to the usual channels frequented by EIP Editors.
+
+##### Quorum
+
+Initially, to establish a valid quorum, all EIP Editors must express their opinion (or vote where appropriate) on the matter under consideration.
+
+After thirty days from the call for input, if not all EIP Editors have responded, the quorum is reduced to the Editors that have responded. This deadline may be extended in exceptional situations.
+
+#### Deciding
+
+##### Electing a Janitor
+
+Any EIP Editor can call for an election for Janitor. The EIP Editor with the most votes once quorum is met is named Janitor until the next election completes. If there is a tie, we'll randomly choose between the EIP Editors with the most votes. Business continues as usual while the election is running.
+
+##### Adding an EIP Editor
+
+An EIP Editor is added once quorum is met, provided no current EIP Editor objects.
+
+##### Removing an EIP Editor
+
+An EIP Editor is removed once quorum is met, provided no current EIP Editor (aside from the one being removed) objects. If the removed Editor was also the Janitor, an election for a new Janitor begins immediately.
+
+##### Other Decisions
+
+All other decisions are made through a "rough consensus" process. This does not require all EIP Editors to agree, although this is preferred. In general, the dominant view of the Editors shall prevail. Dominance, in this process, is not determined by persistence or volume but rather a more general sense of agreement. Note that 51% does not mean "rough consensus" has been reached, and 99% is better than rough. It is up to the Janitor to determine if rough consensus has been reached. Every EIP Editor is entitled to have their opinion heard and understood before the Janitor makes that determination.
+
+No one, not the EIP Editors and certainly not the Janitor, holds veto powers (except when adding/removing an Editor as defined above.) It is imperative that the EIP process evolve, albeit cautiously.
```

Comment:
if janitor decides what is `rough consensus` in contentious issues, he/she effectively has veto powers?

Reply:
well by my interpretation, what the janitor has in this document is more like the authority to defer to a supermajority in overriding an individual's veto. If I may pull an analogy from the USian interpretation of montesquieu, the president can only veto an act of congress if congress votes with less than 2/3 majority.

In a case where one or two editors digs in heels and want to keep discussing, but the rest of the editors feel further deliberation won't be helpful, the difference between "full consensus" (i.e. effective unanimity) and rough consensus (which a janitor could declare) would be more like timeboxing consensus and officially declaring a default to supermajority. It's called "rough" consensus, cuz it can get a little rough!

Reply:
help making this clearer in the document would be appreciated, tho.  I think it's strategically imperative that the document promise no vetos so that if people *feel* vetoed, they can feel empowered to ask for an explanation of what's happening and how it's not as simple as a veto-- so being crystal clear on failure modes and interventions is important here.

Reply:
My exact intention here is that the Janitor can determine _when_ rough consensus has been reached (i.e. ending discussion) but cannot determine _what_ the consensus is.

If a Janitor refuses to declare that consensus has been reached, they can be removed.

Reply:
by deciding `when` it determines the `what`

Reply:
> by deciding `when` it determines the `what`. may be this point needs to be more chiseled out

So you're suggesting the Janitor might declare consensus has been reached before discussion naturally would've ended?

Reply:
i guess so, but then he can be removed so i guess i am good with it